control.conf

```diff
@@ -0,0 +1,7 @@
+[Server]
```

Comment:
This should be `control.toml.example`, and `control.toml` should be `.gitignore`d, so we change change our local config, without the risk of it showing up in git (this trick taken from rustc's build system)

Reply:
Sure

Reply:
Fixed in 7e12fd7

Reply:
Fixed in 762600c